### ISSUE TITLE
Customize widgets: Undo and Redo buttons are disabled thus breaking expected keyboard interaction

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#57678]
### Description

See https://github.com/WordPress/gutenberg/issues/57647

I'm not sure what the plans are for the `customize-widgets` package so I'm not sure it is worth fixing it. However, the `NavigableToolbar` with Undo and Redo buttons suffers from the same bugs reported in https://github.com/WordPress/gutenberg/issues/57647.

- The buttons used `disabled` while they should use `aria-disabled` instead.
- As such, arrows key navigation doesn't work when these buttons are disabled.
- Focus style is not visible.

Screenshot:

![Screenshot 2024-01-09 at 14 51 20](https://github.com/WordPress/gutenberg/assets/1682452/affaf635-184a-411d-a1bc-491643bf8adf)

### Step-by-step reproduction instructions

Basically, repeat the steps from https://github.com/WordPress/gutenberg/issues/57647 but test the Customizer instead.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes
